### PR TITLE
Integrations tab for easy access

### PR DESCRIPTION
**Pull request by SmritiSatyanV**
Adding 'integrations' tab so that users can easily refer to it

### CODE REVIEW

**Comment by pradyunsg** [comment]
Hi there! Look like you filed a PR for your fork, onto the wrong repository. :)

**Comment by SmritiSatyanV** [comment]
Sorry about that. Deleted it.

File: src/furo/theme/furo/base.html [hunk 1]
```diff
@@ -68,12 +68,31 @@
...
+      gtag('config', 'G-YQL24L5CKY', {
+        "custom_map": {
+          "dimension0": "event_category",
+          "dimension1": "event_label",
+          "dimension2": "suggestion_feedback",
+          "dimension3": "rtd_project",
+          "dimension4": "rtd_version"
+        }
+      });
+    </script>
```
Comment (pradyunsg) [inline src/furo/theme/furo/base.html:95]: FYI: You've publicly listed your FontAwesome kit credentials.

**Comment by SmritiSatyanV** [comment]
I have just begun working on it, my understanding is weak, working towards
getting better.

**Pull request closed without merging**